src/extension.ts

```diff
@@ -582,7 +594,9 @@ located next to Run / Debug Tests toolbar buttons.`);
   }
 
   private _testMessageForTestError(testItem: vscodeTypes.TestItem, error: TestError): vscodeTypes.TestMessage {
-    const text = error.stack || error.message || error.value!;
+    let text = error.stack || error.message || error.value!;
+    if (text.includes('Looks like Playwright Test or Playwright'))
+      text = `Browser was not installed. Invoke 'Install Playwright Browsers' action to install missing browsers.`;
```

Comment:
will `Install Playwright Browsers` actually match the action ?

Reply:
Yes